compiler/front/options.nim

```diff
@@ -95,6 +95,8 @@ type
     writtenSemReports*: ReportSet
     lastError*: TLineInfo
     filenameToIndexTbl*: Table[string, FileIndex]
+    rawPathToIndexTbl*: TableRef[string, FileIndex] ## maps non-canonicalized paths 
+    ## of known-files to the corresponding file index
```

Comment:
```suggestion
    rawPathToIndexTbl*: Table[string, FileIndex] ## maps non-canonicalized
    ## paths of known-files to the corresponding file index
```
I've missed a `TableRef` being used in my previous review, apologies. There's no need for using one here.